WordPressEditor/src/main/res/values/colors.xml

```diff
@@ -10,5 +10,7 @@
     <color name="format_bar_button_normal_color">@color/wp_gray</color>
     <color name="format_bar_button_highlighted_color">@color/wp_blue</color>
 
+    <color name="placeholder_text">@color/wp_gray</color>
```

Comment:
Just thinking out loud...do you think we need to be more specific with the name here since this value defines the placeholder color in the source view and not the visual view?